startstoptorq.sh

```diff
@@ -0,0 +1,275 @@
+# Load the environment
+. ./setenv.sh
+
+# sets the base port for a default TorQ installation
+export KDBHDB=${PWD}/hdb/database
+export KDBWDB=${PWD}/wdbhdb
+export KDBSTACKID="-stackid ${KDBBASEPORT}"
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$KDBLIB/l32
+export DEFAULTCSV=${KDBAPPCONFIG}/process.csv
+
+getfield() {
+  fieldno=`awk -F, '{if(NR==1) for(i=1;i<=NF;i++){if($i=="'$2'") print i}}' $csvpath` 		    # get number for field based on headers
+  fieldval=`awk -F, '{if(NR == '$1') print $'$fieldno'}' $csvpath`                    		    # pull one field from one line of file
+  echo $fieldval | envsubst                                                                         # substitute env vars
+ }
+
+parameter() {
+  fieldval=`getfield $1 $2`
+  if [ "" == "$fieldval" ]; then                                                                    # check for empty string
+    echo ""
+  else
+    echo " -"$2 $fieldval
+  fi
+ }
+
+getqcmd() {
+  fieldval=`getfield $1 $2`
+  if [ "" != "$fieldval" ]; then
+    echo $fieldval
+  fi
+ }
+
+findproc() {
+  pgrep -f "\-procname $1 $KDBSTACKID \-proctype $(getfield $1 proctype)"
+ }
+
+startline() {
+  procno=`awk '/'$1'/{print NR}' $csvpath`							    # get line number for file
+  qcmd=`awk '/'$1'/{print NR}' $csvpath`
```

Comment:
is this used for anything? doesn't seem like it is, and it's the same as `procno` so surely if you do need it you can just use that?